keps/sig-architecture/960-conformance-behaviors/README.md

```diff
@@ -242,7 +471,33 @@ type Behavior struct {
 }
 ```
 
-### Behavior and Test Generation Tooling
+#### Generating Lists of Behaviors (Phase 1)
+
+In Phase 1, a `kubetestgen` tool was created that can be used to generate
```

Comment:
Do we have kubetestgen or kubeconform PRs to look at?

Reply:
Yes: https://github.com/kubernetes/kubernetes/pull/83964 and https://github.com/kubernetes/kubernetes/pull/89393. We have some usage examples in the README